recovery_points_for_virtual_server.rb

```diff
@@ -77,4 +80,16 @@ def get_vm_recovery_points(resource_id)
     vm_points.is_a?(Hash) ? [vm_points] : vm_points
   end
 
+  def get_backup_size(backup_id)
```

Comment:
should we protect from the cases where veem is of version which doesn't support this API call?

Reply:
you mean return 0 if smth went wrong?

Reply:
something like that

Reply:
make sense, added default 0 to fetch

Reply:
`&.fetch` will only protect against empty array returned by the GET request, but how does legacy veeam that does not support to this call react to it? Could it raise exception?

Reply:
Guys, please take a look one more time
I've added rescue RestClient::BadRequest in order to still have 0 instead of fail
and also now I downcase resulted hash keys, because in 10th version we have CreationTimeUtc and in 11th - CreationTimeUTC

@binarycode @hasrthur @ykohut